lightningd/test/run-find_my_path.c

```diff
@@ -13,6 +13,9 @@ void db_begin_transaction_(struct db *db UNNEEDED, const char *location UNNEEDED
 /* Generated stub for db_commit_transaction */
```

Comment:
I usually roll these together, to avoid breaking bisect.

We could actually dynamically generate these anyway, by doing `#include <gen_mocks_xxx>` and generating those gen_mocks files.  That would avoid these issues altogether.

Reply:
Hm, that was my intention as well, hence I committed the stubs before committing the changes requiring them. I'll merge the two commits :+1: